stable/gangway/README.md

```diff
@@ -74,6 +74,8 @@ At a minimum you *must* configure any of the values marked as **required** in th
 | `podAnnotations`                 | Additional annotations to apply to the pod.                                                                                                                                                                     | `{}`                                               |
 | `resources`                      | CPU/Memory resource requests/limits.                                                                                                                                                                            | `{}`                                               |
 | `readinessProbe.scheme`          | Scheme to use for httpGet probe, `HTTP` or `HTTPS`.                                                                                                                                                             | `HTTP`                                             |
+| `serviceAccount.create`                   | Determines if a different serviceAccount should be used other than `default`. Requires `serviceAccount.name` to be set. on                                                                                                                                                                           | `false`                                               |
+| `serviceAccount.name`                   | Sets the name of the serviceAccount to use for Gangway. Requires `serviceAccount.create` to be set to `true`. on                                                                                                                                                                           | `gangway`                                               |
```

Comment:
Could probably avoid having the one above as well, and simply detect the presence of this option for the creation. Minor point though.

Reply:
@rk295 , thanks for the suggestion. Would it be recommended that I remove line 78 and change line 77 to:

```
| `serviceAccount.create`                   | Determines if an alternative serviceAccount should be created and used other than `default`. Requires `serviceAccount.name` to be set (defaults to `gangway`)                                                                                                                                                                           | `false` 
```

Reply:
I’d probably drop the create line and have the service account name field. The. It’s explicit what it is. Then you of just checks for the empty string.

Reply:
ok I see. I'll make the change and test @rk295. Thanks!